app/src/main/java/com/github/wnder/NetworkInformation.java

```diff
@@ -0,0 +1,20 @@
+package com.github.wnder;
+
+import android.content.Context;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+
+import static androidx.core.content.ContextCompat.getSystemService;
+
+public class NetworkInformation {
+    public static boolean isNetworkAvailable(Context ctx) {
+        ConnectivityManager manager = (ConnectivityManager) ctx.getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo networkInfo = manager.getActiveNetworkInfo();
+        boolean isAvailable = false;
+        if (networkInfo != null && networkInfo.isConnected()) {
```

Comment:
Just want to make sure: are you sure this verifiy any type of internet connecivity? WiFi and cellular?

Reply:
Okay thanks